scripts/generate.sh

```diff
@@ -69,6 +69,11 @@ for tags in $($GETTAGS "${FILTER}"); do
             cp "${dir}/run" "${imgdir}/run"
             chmod 755 "${imgdir}/run"
             RUN="RUN mkdir -p /etc/service/${SERVICE}\nCOPY run /etc/service/${SERVICE}/run"
+            if [ -e "${dir}/finish" ]; then
```

Comment:
It seems it would be much more legible if this $dir variable were named $services_dir or $service_dir. But that's a _refactoring_ not for the faint of heart at this point.